### ISSUE TITLE
Feature Request - Menu should be available when viewing pics in "Gallery / Photos" tab

### ISSUE THREAD

**Issue by telrod11** [FilenCloudDienste/filen-mobile#219]
Go to **Photos** tab
Click a **thumbnail** to bring up **full** size

Here I should have access to the full menu, just as it is available with the thumbnails

Be curious to see if this works on the IOS side, if anyone can comment.

Full menu
![Screenshot_20230419_071737_Filen](https://user-images.githubusercontent.com/105058921/233059998-48ddb58f-f4f6-4647-8b5a-6a576e88b37c.jpg)

**Comment by Dwynr** [comment]
I agree, but I've removed your sample image from the issue due to privacy reasons.

**Comment by telrod11** [comment]
> I agree, but I've removed your sample image from the issue due to privacy reasons.

Thanks, I appreciate that.   Overkill for me to have put up the menu that everyone knows anyway...  :)

**Comment by skauertz** [comment]
If you mean the menu that has Share / Download / Save to Gallery and so on, on iOS it doesn't let you access that on full-screen images either (would be surprising if it did on one platform and not the other actually).
